widget_driver/test/drivable_widget_test.dart

```diff
@@ -66,5 +67,90 @@ void main() {
 
       expect(find.text('didCallTestMethod: true'), findsOneWidget);
     });
+    testWidgets('On state change updateDriverProvidedProperties gets called', (WidgetTester tester) async {
+      when(() => _mockRuntimeEnvironmentInfo.isRunningInTestEnvironment()).thenReturn(false);
+
+      final testContainerDrivableWidget = TestContainerDrivableWidget(
+        environmentInfo: _mockRuntimeEnvironmentInfo,
+      );
+      await tester.pumpWidget(MaterialApp(home: testContainerDrivableWidget));
+      final driver = testContainerDrivableWidget.driver;
+
+      expect(driver.updateCount, 1);
+
+      driver.aTestMethod();
```

Comment:
Maybe mention here why you call this `driver.aTestMethod();` method..
Or call some other method on a widget ontop of the drivveable Widget..

I guess you call this so that the `notifyWidgets` gets called which makes the build method get called..

But another way could be if you from a widget on top could make the build method get called..